src/components/singular/StockSelect.js

```diff
@@ -0,0 +1,193 @@
+import { React, useState, useEffect, useRef, useCallback } from "react";
```

Comment:
We shouldn't have a new component. The purpose of creating the reusbale search component is so we wouldn't ave to do it this.